crud/select/compat/common.lua

```diff
@@ -1,6 +1,32 @@
+local ratelimit = require('crud.ratelimit')
+local check_select_safety_rl = ratelimit.new()
```

Comment:
For now we don't set any limits on warnings count, aren't we?

Reply:
Default values are used (as in the Tarantool implementation).

Reply:
Ok, thanks